eldritch/stories.py

```diff
@@ -0,0 +1,402 @@
+  def __init__(self):
+    super().__init__("Final Analysis", "Cured", "Followed")
+
+  def get_trigger(self, event, owner, state):
+    if isinstance(event, events.GainOrLoss) and owner.clues >= 6:
+      return self.advance_story(owner, False)
+    return None
+
+  def get_usable_interrupt(self, event, owner: characters.Character, state):
+    if (
+        isinstance(event, events.Upkeep)
+        and event.character == owner
+        and owner.place.name == "Asylum"
+    ):
+      choice = events.SpendChoice(
+          owner,
+          "Spend $15 to finish story?",
+          ["No", "Yes"],
+          spends=[None, values.ExactSpendPrerequisite({"dollars": 15})]
+      )
+
+      result = events.Conditional(owner, choice, "choice_idx", {1: self.advance_story(owner, True)})
+      return events.Sequence([choice, result], owner)
+    return None
+
+
+class SalesmanStoryPass(StoryResult):
+  def __init__(self):
+    super().__init__("Jackpot")
+
+  def get_in_play_event(self, owner: characters.Character):
+    return events.Gain(owner, {"dollars": 15}, self)
+
+  def get_usable_interrupt(self, event, owner, state):
+    if (
+        isinstance(event, events.Upkeep)
+        and event.character == owner
+        and not self.exhausted
+    ):
+      choice = events.SpendChoice(
+          owner, "Spend $3 for 1 clue token?",
+          ["No", "Yes"],
+          spends=[None, values.ExactSpendPrerequisite({"dollars": 3})]
+      )
+      cond = events.Conditional(owner, choice, "choice_idx", {1: events.Gain(owner, {"clues": 1})})
+      return events.Sequence([choice, cond], owner)
+    return None
+
+
+class SalesmanStoryFail(StoryResult):
+  def __init__(self):
+    super().__init__("Greed")
+
+  def get_in_play_event(self, owner: characters.Character):
+    return events.Curse(owner)
+
+  def get_interrupt(self, event, owner: characters.Character, state):
+    if isinstance(event, events.Bless) and event.character == owner and owner.dollars > 0:
+      return events.CancelEvent(event)
+    if (
+        isinstance(event, events.GainOrLoss)
+        and event.character == owner
+        and event.gains.get("dollars", 0) > 0
+        and owner.bless_curse == 1
+    ):
+      return events.Curse(owner)
+    return None
+
+
+class SalesmanStory(Story):
+  def __init__(self):
+    super().__init__("Old Money", "Jackpot", "Greed")
+
+  def get_trigger(self, event, owner: characters.Character, state):
+    if owner.clues >= 5:
+      return self.advance_story(owner, True)
+    if len([t for t in owner.trophies if isinstance(t, monsters.Monster)]) >= 3:
+      return self.advance_story(owner, False)
+    return None
+
+
+class ScientistStoryPass(StoryResult):
+  def __init__(self):
+    super().__init__("It's Working!")
+
+  def get_modifier(self, other, attribute):
+    if attribute == "gate_limit":
+      return 1
+    return 0
+
+  def get_in_play_event(self, owner: characters.Character):
+    return events.AddGlobalEffect(self)
```

Comment:
Does this need to be a global effect? I thought cards in users' possessions would automatically be taken into account for global modifiers?

Reply:
```python
  def globals(self) -> List[mythos.GlobalEffect]:
    return [self.rumor, self.environment, self.ancient_one] + self.other_globals
```
Looks to me like characters possessions aren't usually checked.

Reply:
That's a TODO for me